Applications/Internet/Internet Explorer 7.0/Online/script.js

```diff
@@ -10,232 +10,311 @@ include("engines.wine.verbs.sandbox");
 include("engines.wine.plugins.windows_version");
 
 var installerImplementation = {
```

Comment:
I couldn't change this script to the new style because it doesn't use a quick script. @plata @ImperatorS79 do you have an idea how we could transform this to an JS object?